tests/Feature/GeneratesTypesCommandTest.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use Exception;
+use Tests\TestCase;
+use Illuminate\Support\Str;
+
+/**
+ * @internal
+ */
+class GeneratesTypesCommandTest extends TestCase
+{
+    /**
+     * @var string[]
+     */
+    protected $modelList = ['bar', 'foo', 'foo-bar'];
+
+    /**
+     * @test
+     * @throws Exception
+     */
+    public function command_absolute_path(
+    ) {
+        $this->runCommand();
+    }
+
+    /**
+     * @test
+     * @throws Exception
+     * @return void
+     */
+    public function command_option_namespace()
+    {
+        $this->runCommand('--namespace=Tests\\Models');
+    }
+
+    /**
+     * @test
+     * @throws Exception
+     * @return void
+     */
+    public function command_option_no_kebab_case()
+    {
+        $this->runCommand('--noKebabCase');
+    }
+
+    /**
+     * @test
+     * @throws Exception
+     * @return void
+     */
+    public function command_option_model()
+    {
+        $this->modelList = ['foo'];
+        $modelPath = addslashes('--model=Tests\\Models\\Foo');
+
+        $this->runCommand($modelPath);
+    }
+
+    /**
+     * @test
+     * @throws Exception
+     * @return void
+     */
+    public function not_a_model()
+    {
+        $modelDir = __DIR__ . '/../Models';
+        $tempFile = $modelDir . '/TestTrait.php';
+        $tempFileContent = <<<'EOD'
+<?php
+
+namespace Tests\Models;
+
+trait TestTrait {
+    public function test() {
+        dd('test');
+    }
+}
+EOD;
```

Comment:
Is het niet cleaner om de trait gewoon standaard in de models directory te zetten?